test/puppetlabs/puppetdb/testutils/reports.clj

```diff
@@ -61,12 +61,26 @@
               (dissoc :environment :containing_class :certname)))
         resource-events)))
 
+(defn munge-resources-for-comparison [resources]
+  (let [resources-sort-fn (partial sort-by
+                                   (fn [r]
+                                     (mapv r [:resource_type :resource_title :timestamp])))
+        events-sort-fn (partial sort-by
+                                (fn [e]
+                                  (mapv e [:property :old_value :new_value])))]
```

Comment:
I thought the uniqueness here was for property only, I didn't think you could multiple changes on a property within one report.